programs/pkey/pk_decrypt.c

```diff
@@ -59,7 +62,8 @@ int main( void )
 int main( int argc, char *argv[] )
 {
     FILE *f;
-    int ret, c;
+    int ret = 1, c;
```

Comment:
Any particular reason for initialising `ret` with `1` that indicates error. It gives the impression that it would be set to `0` at the end. Perhaps it is a leftover from a previous change. The first file in this PR has it as `ret = 0`. Though it does not have an effect on program behaviour, it is slightly distracting.

Reply:
It is a hand over from a previous change, but as you pointed out it does not make a difference because (1) the value of ret is never checked before assigning the return value of a function and (2) the compiler would optimize dead code anyways.